d2renderers/d2svg/d2svg.go

```diff
@@ -701,17 +702,27 @@ func drawShape(writer io.Writer, targetShape d2target.Shape) error {
 			}
 			fmt.Fprintf(writer, "</g></g>")
 		case d2target.ShapeText:
-			render, err := textmeasure.RenderMarkdown(targetShape.Label)
-			if err != nil {
-				return err
+			if targetShape.Language == "latex" {
+				render, err := d2latex.Render(targetShape.Label)
+				if err != nil {
+					return err
+				}
+				fmt.Fprintf(writer, `<g transform="translate(%f %f)" style="opacity:%f">`, box.TopLeft.X, box.TopLeft.Y, targetShape.Opacity)
```

Comment:
```suggestion
				fmt.Fprintf(writer, `<g transform="translate(%f, %f)" style="opacity:%f">`, box.TopLeft.X, box.TopLeft.Y, targetShape.Opacity)
```

Reply:
that's not right: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform#translate